modules/ztp-image-based-upgrade-talm.adoc

```diff
@@ -0,0 +1,375 @@
+// Module included in the following assemblies:
+// Epic TELCOSTRAT-160 (4.15/4.16), story TELCODOCS-1576
+// * scalability_and_performance/ztp-talm-updating-managed-policies.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-image-based-upgrade-with-talm_{context}"]
+= Upgrading the {sno} cluster with {lcao} through GitOps ZTP
+
+You can upgrade your managed {sno} cluster with the image-based upgrade through ZTP GitOps.
+//draft
+
+.Prerequisites
+
+* {rh-acm} minimum version
+* The target cluster is bound to existing ZTP policies.
+* You generated a seed image from a compatible seed cluster.
+//draft
+
+.Procedure
+
+. Create a policy for the OADP `ConfigMap`, named `oadp-cm-common-policies`. For more information about how to create the `ConfigMap`, follow the first step in _Preparing the single-node OpenShift cluster for the image-based upgrade_ in _Additional resources_.
+
+. Create a `PolicyGenTemplate` CR for the `ImageBasedUpgrade` CR for the `Prep` stage:
++
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1
+kind: PolicyGenTemplate
+metadata:
+  name: group-ibu
+  namespace: "ztp-group"
+spec:
+  bindingRules:
+    group-du-sno: ""
+  mcp: "master"
+  sourceFiles:
+    - fileName: ImageBasedUpgrade.yaml
+      policyName: "prep-policy"
+      metadata:
+        name: "upgrade"
+      spec:
+        stage: Prep <1>
+        seedImageRef: <2>
+          version: "4.16.0"
+          image: "quay.io/user/upgbackup:lca-seed"
+          pullSecretRef:
```

Comment:
should the `pullSecretRef` be documented earlier in one of the IBU CRs? this is the first it is mentioned, but would be handy earlier, even for people not using GitOps to run IBU

Reply:
Good catch! I updated the [Prep IBU CR](https://69809--ocpdocs-pr.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp_far_edge/ztp-talm-updating-managed-policies#ztp-image-based-upgrade-prep_ztp-talm) (will be in step 2 example once the preview is updated)